components/contextualhelp/metadata/contextualhelp.yml

```diff
@@ -0,0 +1,61 @@
+name: Contextual Help
+status: Verified
+SpectrumSiteSlug: https://spectrum.adobe.com/page/contextual-help/
+sections: 
+- name: Custom Properties API
+  description: |
+    This component can be modified via its `--mod-*` prefixed custom properties. A list of those prefixed custom properties can be found <a href="https://github.com/adobe/spectrum-css/tree/main/components/popover/metadata/mods.md">here</a>.
+examples:
+  - id: contextualhelp-variants
+    name: Info Icon
+    markup: |
+      <div class="spectrum-Examples">
+        <div class="spectrum-Examples-item">
+          <div class="spectrum-ContextualHelp">
+            <button class="spectrum-ActionButton spectrum-ActionButton--sizeXS spectrum-ContextualHelp-button">
+              <svg class="spectrum-Icon spectrum-Icon--sizeM spectrum-ActionButton-icon" focusable="false" aria-hidden="true">
+                <use xlink:href="#spectrum-icon-18-Info" />
+              </svg>
+            </button>
+            <div class="spectrum-Popover is-open spectrum-Popover--bottom-start spectrum-ContextualHelp-popover" role="presentation">
+              <h2 class="spectrum-ContextualHelp-heading">Permission required</h2>
+              <p class="spectrum-ContextualHelp-body">Cupcake ipsum dolor sit amet jelly beans. Chocolate jelly caramels. Icing soufflé chupa chups donut cheesecake. Jelly-o chocolate cake sweet roll cake danish candy biscuit halvah.</p>
+            </div>
+            <div class="dummy-spacing" style="position: relative; height: 248px; min-width: 400px; max-width: 50%;"></div>
+          </div>
+        </div>
+        <div class="spectrum-Examples-item">
+          <div class="spectrum-ContextualHelp">
```

Comment:
wondering if each of these should be separate variants instead of displayed together? If the intent is to allow copy and paste of the components with the displayed markup I think It would make more sense for them to be separate variants.

Reply:
I can separate them out if you think that would be best. I was kind of following the layout of Popover https://opensource.adobe.com/spectrum-css/popover.html

Reply:
ah, I had not looked at popover before and I see how thats similar, I'd default to what @castastrophe thinks.